course/layouts/index.coursedata.json

```diff
@@ -0,0 +1,34 @@
+{{- $courseData := .Site.Data.course -}}
+{{- $instructors := slice -}}
+{{- if not (eq  $courseData.instructors nil) -}}
+{{- $instructors = partial "get_instructors.html" $courseData.instructors.content -}}
+{{- end -}}
+
+{
+  "course_title": {{- $courseData.course_title | jsonify -}},
+  "course_description": {{- $courseData.course_description | jsonify -}},
+  "site_uid": {{- $courseData.site_uid | jsonify -}},
+  "legacy_uid": {{- $courseData.legacy_uid | jsonify -}},
+  "instructors": [
+  {{- range $index, $value := $instructors -}}
```

Comment:
Bit of a nitpick, but instead of `$value` here can we use something more descriptive like `$instructor`?